platform/packages/finance/src/coin/dto/mod.rs

```diff
@@ -28,8 +28,11 @@ mod unchecked;
 /// carries also the currency ticker. The aim is to use it everywhere the cosmwasm
 /// framework does not support type parameterization or where the currency type
 /// is unknown at compile time.
-#[derive(Clone, PartialEq, Eq, Debug, Serialize, Deserialize, JsonSchema)]
-#[serde(try_from = "unchecked::CoinDTO")]
+#[derive(PartialEq, Eq, Debug, Serialize, Deserialize, JsonSchema)]
+#[serde(
+    try_from = "unchecked::CoinDTO",
+    bound(serialize = "", deserialize = "")
```

Comment:
what bounds do we override with this?

Reply:
As you might have noticed, without this override, the derived versions place additional bounds on all generics. Since currencies and groups are logical constructs, not functional ones, this code works exactly as we'd expect it to without them implementing, as you liked to refer to it when we talked about `#[cfg_attr(..., derive(Debug))]`, unnecessary code. If we are not making exceptions for Debug, whose implementation is dropped once linker strips unused symbols, then this is more than unacceptable.